keyring-util/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "keyring-util"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-std = { version = "1.12.0", features = [ "attributes" ] }
+chrono = "0.4.31"
+clap = { version = "4.4.6", features = [ "cargo", "derive" ] }
+oo7 = { path = "../" }
+rpassword = "7.2.0"
+zbus = { version = "3.5.0", default-features = false}
```

Comment:
why do we need direct zbus dependency when it is re-exported by oo7?

Reply:
It is not, but I added it.